docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -12,17 +12,33 @@ and bronchiolitis, is a major global killer of both children and adults. Symptom
 include shortness of breath, weakness, fever, coughing and fatigue. It is important to check for a fever. Symptoms can last about 7 days and the infection is contagious
 to others shortly before and while experiencing symptoms. It is mainly
 caused by four microorganisms - Streptococcus pneumoniae (*pneumococcal
```

Comment:
Here microorganisms = pathogens, yeah? Maybe just call them pathogens to be consistent?

Reply:
Thanks for the comment. I will make sure it is consistent.